ingest/api/dssapi.py

```diff
@@ -77,6 +85,7 @@ def put_file(self, bundle_uuid, file):
                 bundle_file = None
 
                 if bundle_uuid:
+                    self.init_dss_client()
```

Comment:
Instead of instantiating in each conditional (line 88 + line 97), you could instantiate once at the beginning of the try block. Perhaps right after the logging line (line 85).